manifests/0000_51_olm_02_operator_clusterrole.yaml

```diff
@@ -40,6 +40,9 @@ rules:
       - list
       - watch
       - create
+      - update
+      - patch
+      - delete
```

Comment:
> If we can create it, we can patch, update, delete
>
> Except events, clusteroperators: we can't delete

I can see "if we can `create`, we can `update` and `patch`".  I'm less clear on `delete`.  Maybe you can put that off until you have operator code-paths you can point to that actually attempt to delete resources?  Because `create`, the update verbs, and the view verbs give you enough to manage a forever-child, and you only need `delete` if you decide you no longer need that child resource.  And global ConfigMap `delete` (and similar for the other `resources`) seems like a pretty broad power, that is worth doing without unless you actually need that power.

Reply:
That's fair, we will need delete at some point, but it can be limited in scope.